<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPR</class>
 <widget class="QMainWindow" name="TestPR">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>688</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TestPR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>688</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>644</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>131</width>
      <height>688</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>688</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: rgb(46, 52, 54);
	color: #FFFFFF;
}

QPushButton:pressed {
	background-color: #FF8C00;
}</string>
    </property>
    <widget class="QPushButton" name="addFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Add File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addFunction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>145</y>
       <width>131</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add Function</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addTest">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>131</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>Add a test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genMake">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>435</y>
       <width>131</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Generate
makefile</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>500</y>
      <width>871</width>
      <height>188</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: #FF8C00;
}

QPushButton {
	background-color: #FFD700;
	color: #2E3436;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>871</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
